cpp/proto/arcticc/pb2/descriptors.proto

```diff
@@ -20,7 +20,7 @@ message TypeDescriptor {
         FLOAT = 3;
 
         BOOL = 4;
-        MICROS_UTC = 5;
+        NANOSECONDS_UTC = 5;
```

Comment:
I think we've seen before that enum names can be changes without issue so I think this should be fine - but to be safe we should test that:

- This enables you to read data written prior to this change
- Older clients can read data written after this change has taken effect

Reply:
Ah, I see this is being run through the regressions tests. In which case we'd just need to test the latter case I think.

Reply:
Reading [the documentation](https://protobuf.dev/programming-guides/encoding/#structure), it is changeable if the fields' ordering does not change:

> The binary version of a message just uses the field’s number as the key – the name and declared type for each field can only be determined on the decoding end by referencing the message type’s definition (i.e. the .proto file). Protoscope does not have access to this information, so it can only provide the field numbers.

If I am not mistaken, @poodlewars proposed that he runs extra checks to ensure that the _required verifications_ that you propose are verified.

Reply:
Yep. @mehertz is still fixing the internal build. Once that's done we can validate these changes against old data.

Reply:
Thanks for the update.